src/components/NavigationDrawer/index.js

```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from 'react';
 import { Link } from 'dva/router';
 import { PageSidebar, Nav, NavList, NavItem } from '@patternfly/react-core';
 import getMenuData from '../../common/menu';
+import './index.less';
```

Comment:
There is no `index.less` now and the padding is still there

Reply:
@aquibbaig I somehow missed adding the file to commit. Please re-review it now.